snowplow_tracker/emitter_configuration.py

```diff
@@ -53,11 +53,11 @@ def __init__(
         :type request_timeout:  float | tuple | None
         """
 
-        self.buffer_size = buffer_size
-        self.on_success = on_success
-        self.on_failure = on_failure
-        self.byte_limit = byte_limit
-        self.request_timeout = request_timeout
+        self._buffer_size = buffer_size
+        self._on_success = on_success
+        self._on_failure = on_failure
+        self._byte_limit = byte_limit
+        self._request_timeout = request_timeout
```

Comment:
I don't think we should set directly to the private properties here because then we skip the validations that are in the public property setters.